tests/public-dns-tests/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "public-dns-tests"
```

Comment:
I think it might make sense to just name this `extended-dns-errors`, instead of a faux-generic collection -- having more crates here is relatively cheap, and changing the name later is, too.